languages/java/exercises/concept/booleans/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
you can remove pom.xml. We use only gradle as build system

Reply:
Done